webui/react/src/components/ResourcePoolBindingModal.tsx

```diff
@@ -40,6 +42,10 @@ const ResourcePoolBindingModalComponent: React.FC<Props> = ({
         targetListTitle={`Bound to ${pool}`}
         onChange={setVisibleBindings}
       />
+      <Paragraph>
+        Note: Binding a resource pool to a workspace prevents other workspaces from using it.
```

Comment:
Sorry my bad, one slight change. Can we say Binding a resource pool to a workspace(s) prevents...